lidnd/app/[username]/[campaign_slug]/party/page.tsx

```
@@ -0,0 +1,262 @@
+
+function PartyLevelInput() {
+  const [campaign] = useCampaign();
+  const { mutate: updateCampaign } = useUpdateCampaign(campaign);
+
+  const [partyLevel, setPartyLevel] = React.useState(
+    campaign?.party_level ?? 1
+  );
+
+  const handlePartyLevelChange = useDebouncedCallback((level: string) => {
+    const stringAsInt = parseInt(level);
+    if (!isNaN(stringAsInt)) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)